### ISSUE TITLE
Change of display confuses cursor

### ISSUE THREAD

**Issue by brianharvey** [jmoenig/Snap--Build-Your-Own-Blocks#1733]
I'm watching a class and some kids just finished a demo of their work, and when they unplugged from the projector with a block editor open, afterward each mouse click just moved the editor window around to where the click was.  They had to restart Chrome to fix it.

Since just reloading Snap<i>!</i> wasn't good enough, maybe it''s a Chrome problem instead of ours, but otoh it was a Morph (the editor) that moved around, not the whole canvas.

**Comment by jmoenig** [comment]
unplugging the projector almost certainly changed the display resolution, right? Probably just clicking the mouse button anywhere would have totally solved the issue, I bet.

**Comment by brianharvey** [comment]
Yes about changing the resolution.  No about clicking the mouse; we tried that a lot!

**Comment by jmoenig** [comment]
hmm... did you also try resizing the browser to force a screen-refresh?

**Comment by brianharvey** [comment]
I'm not positive, but I think we tried that too.

Used to be that every so often I'd have a problem in which Snap!'s idea about where the mouse cursor is would be off by an inch or so, and at first I thought this was the same thing, but this was different, because any click anywhere was taken to mean "move the editor window here" as if some modifier bit were stuck on.  Really weird.

**Comment by DarDoro** [comment]
Quite similar behaviour can be reproduced by dragging object to the edge of Snap window then releasing button outside ( for example on browser bar area). Object stays "glued" to mouse pointer.

`HandMorph.prototype.processMouseMove@morphic.js` checks only  `this.mouseButtons` stored at mouseDown & Up instead of current `event.buttons` bit mask.
